examples/network_traffic_prediction/Synthetic_Airport_Traffic_wAirlines.ipynb

```diff
@@ -0,0 +1,1597 @@
+    "from cartopy import crs as ccrs\n",
+    "from cartopy.io import shapereader"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "6a96c307-a582-451e-bdc8-ae7e20fe3ff3",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Set up the total days for generating data\n",
+    "NUM_DEMAND_DAYS = 31\n",
+    "np.random.seed(47)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76ab15dc-00cc-49cf-94cf-c8fb91e8ac9b",
+   "metadata": {},
+   "source": [
+    "## Extract Major Airports in the World"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "ebf608a3-cd8a-4235-8414-7e34dd47536b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Fetch a list of N airport codes (three letter IATA codes) and their lat, lon coordinates\n",
+    "def fetch_airports_data(N=100):\n",
+    "    # You can use a public CSV file with airport data\n",
+    "    path = 'https://ourairports.com/data/airports.csv'\n",
+    "\n",
+    "    airports = pd.read_csv(path)\n",
+    "\n",
+    "    # Process `continent` column to replace NaN with string NA\n",
+    "    airports['continent'].fillna('NA', inplace=True)\n",
+    "\n",
+    "    # filter out valid major airports in the world\n",
+    "    airports = airports[(airports['type'] == 'large_airport') & \\\n",
+    "                        airports['iata_code'].notna() & \\\n",
+    "                        airports['iso_country'].notna()]\n",
+    "\n",
+    "    # Select only the first N airports and relevant columns\n",
+    "    selected_airports = airports[['iata_code', 'name', 'continent', 'iso_country', 'latitude_deg', 'longitude_deg']].head(N)\n",
+    "    return selected_airports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "12b22747-63cb-4e9d-b203-4d397567a59c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(472, 6)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Extract first 1000 major airports\n",
+    "airports_df = fetch_airports_data(N=1000)\n",
+    "\n",
+    "# Turns out there are only 471 major airports\n",
+    "print(airports_df.shape)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "17a4f570-ee2c-4065-9826-306d36fe148f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Create a Networkx graph where nodes are the airports, and edges are demands between any two airports.\n",
+    "def create_airport_graph(airports_df):\n",
+    "    G = nx.Graph()\n",
+    "\n",
+    "    # Add nodes\n",
+    "    for _, row in airports_df.iterrows():\n",
+    "        G.add_node(row['iata_code'], pos=(row['longitude_deg'], row['latitude_deg']))\n",
+    "\n",
+    "    # Create edges between any two airports but not self loop\n",
+    "    np.random.seed(42)  # For reproducibility\n",
```

Comment:
Haven't we set it already?

Reply:
This notebook does not use GraphStorm. Here the random seed is for numpy only.